bundles/framework/coordinatetool/instance.js

```diff
@@ -91,7 +91,7 @@ Oskari.clazz.define("Oskari.mapframework.bundle.coordinatetool.CoordinateToolBun
             me.setSandbox(sandbox);
             this.coordinateToolService = this.createService(sandbox, conf);
             var mapModule = sandbox.findRegisteredModuleInstance('MainMapModule');
-            var locale = this.getLocalization('display');
+            var locale = Oskari.getMsg.bind('display', 'coordinatetool');
```

Comment:
Does 'display' in here do anything? Looks like every other piece of code using Oskari.getMsg.bind passes null as the first parameter. And later on you are still prefixing everything with "display".

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind the first parameter assigns "this" for the function which makes "display" string a weird choice for that.